src/styles/StyleUtils.js

```diff
@@ -45,17 +79,46 @@ function getAvatarStyle(size) {
     };
 }
 
+/**
+* Return the border style if avatar is SVG
+*
+* @param {String} size
+* @param {String} type
+* @returns {Object}
+*/
+function getAvatarBorderRadius(size, type) {
+    if (type === CONST.ICON_TYPE_WORKSPACE) {
+        return {borderRadius: avatarBorderSizes[size]};
+    }
+
+    // Default to rounded border
+    return {borderRadius: variables.buttonBorderRadius};
+}
+
 /**
  * Return the border style if avatar is SVG
  *
- * @param {Boolean} isSVG
+ * @param {String} size
+ * @param {String} type
  * @returns {Object}
  */
-function getAvatarSVGBorder(isSVG) {
-    if (!isSVG) {
-        return {};
-    }
-    return styles.svgAvatarBorder;
+function getAvatarBorderStyle(size, type) {
+    return {
+        overflow: 'hidden',
+        ...getAvatarBorderRadius(size, type),
+    };
+}
+
+/**
+ * Helper method to return old dot default avatar associated with login
+ *
+ * @param {String} [workspaceName]
+ * @returns {Object}
+ */
+function getDefaultWorspaceAvatarColor(workspaceName) {
+    const colorHash = ReportUtils.hashLogin(workspaceName.trim(), workspaceColorOptions.length);
+
+    return {backgroundColor: workspaceColorOptions[colorHash][0], fill: workspaceColorOptions[colorHash][1]};
```

Comment:
NAB: an option would have been to store the colours in the constant `workspaceColorOptions` like this:

```js
const workspaceColorOptions = [
    {backgroundColor: colors.blue200, fill: colors.blue700},
    {backgroundColor: colors.blue400, fill: colors.blue800},
    {backgroundColor: colors.blue700, fill: colors.blue200},
    {backgroundColor: colors.green200, fill: colors.green700},
    {backgroundColor: colors.green400, fill: colors.green800},
    {backgroundColor: colors.green700, fill: colors.green200},
    {backgroundColor: colors.yellow200, fill: colors.yellow700},
    {backgroundColor: colors.yellow400, fill: colors.yellow800},
    {backgroundColor: colors.yellow700, fill: colors.yellow200},
    {backgroundColor: colors.tangerine200, fill: colors.tangerine700},
    {backgroundColor: colors.tangerine400, fill: colors.tangerine800},
    {backgroundColor: colors.tangerine700, fill: colors.tangerine400},
    {backgroundColor: colors.pink200, fill: colors.pink700},
    {backgroundColor: colors.pink400, fill: colors.pink800},
    {backgroundColor: colors.pink700, fill: colors.pink200},
    {backgroundColor: colors.ice200, fill: colors.ice700},
    {backgroundColor: colors.ice400, fill: colors.ice800},
    {backgroundColor: colors.ice700, fill: colors.ice200},
];
```

And then this return changes to

```suggestion
    return workspaceColorOptions[colorHash];
```

but what I'm proposing may be a bad idea if `workspaceColorOptions` will have other uses.

Reply:
I think this is a good idea, I'll do this instead